README.md

```diff
@@ -0,0 +1,165 @@
+- A list of different categories of meals
+Comment button feature
+Like button feature
+- 
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]
+
+https://toyman640.github.io/Javascript-Capstone-project/dist/
+
+## Video Walkthrough 📹🚶
+
+- [Video-Walkthrough] https://drive.google.com/file/d/1wKLEIDIWqFQyXnap5koxtaBsWNyjYJ9a/view?usp=sharing
+
+
+
+<p align="right">(<a href="#readme-top"></a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Web browser.
+- Code editor eg [Visual Studio Code](https://code.visualstudio.com/download).
+- Github: Generate an ssh key or use https to clone the repository.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- With a git cmd, git bash or visual studio terminal, clone repo with **``git@github.com:toyman640/Javascript-Capstone-project.git``** to download the repository locally on your machine.
+- Then you navigate to the folder in your code editor with the command *``cd Javascript-Capstone-project``*.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Maurice Coker**
+
+- GitHub: [@coker2019](https://github.com/coker2019)
+- Twitter: [@Olabisi_coker](https://twitter.com/Olabisi_coker)
+- LinkedIn: [maurice-coker-174b129b](https://linkedin.com/in/maurice-coker-174b129b)
+
+ 👤 **Falako Omotoyosi**
+
+- GitHub: [@toyman640](https://github.com/toyman640)
+- Twitter: [@_toyman](https://twitter.com/_toyman)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+- Links to different pages will be add
+ CSS styling will be added to the page to make it look more professional
+-
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+Show your support ⭐️
+If you like this project, please give it a star and share it with your friends.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Microverse Community for inspiring this project and documentations. It is a pleasure to have joined the program and I hope to learn a lot more as I progress.
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ Required ] I think it would be better if you fixed the LICENSE link on your readme so that it doesn't lead to a `404` page not found error. Currently, the license link links to a file that does not exist `MIT.md`  in your directory thus leading to the  `404` page not found error. I would like to suggest that you change the link to the one below for the Link to work as expected.

```sh
This project is [MIT](./LICENSE) licensed.
```